src/validation.cpp

```diff
@@ -1153,9 +1154,9 @@ CChainState::CChainState(
     BlockManager& blockman,
     ChainstateManager& chainman,
     std::optional<uint256> from_snapshot_blockhash)
-    : m_mempool(mempool),
-      m_blockman(blockman),
+    : m_blockman(blockman),
```

Comment:
Is there any particular reason for reordering the initialiser list?

Reply:
Member variables must be initialized in the order in which they are declared. Otherwise, CI will throw `[-Werror = reorder]`.
See https://stackoverflow.com/questions/12222417/why-should-i-initialize-member-variables-in-the-order-theyre-declared-in

Reply:
Thanks!